toptal/picasso
# feat(Notification): add notification actions

[FX-210](https://toptal-core.atlassian.net/browse/FX-210)

### How to test

- Navigate to `Notification` section
- Observe new sub-section `Notification.Actions`
- Observe new example `With Action`

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| N/A | ![image](https://user-images.githubusercontent.com/9822091/66553553-a08a3f00-eb4b-11e9-9f8e-f318b5c37402.png) |

### Review
